ndcube/visualization/descriptor.py

```diff
@@ -28,12 +32,18 @@ def __get__(self, obj, objtype=None):
                 try:
```

Comment:
You could probably save some lines here by putting the if block inside the the try block?